apps/server/src/modules/payment/stripe.ts

```diff
@@ -0,0 +1,18 @@
+import { FactoryProvider } from '@nestjs/common';
+import { omit } from 'lodash-es';
+import Stripe from 'stripe';
```

Comment:
Do you need to install it? I don't see dep added in package.json.

Reply:
installed in target PR already, need some startup changes 🐸